man/kernel-command-line.xml

```diff
@@ -90,6 +91,16 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>systemd.early_core_pattern=</varname></term>
+        <listitem>
+          <para>During early boot, the generation of core dump files is disabled until a core dump handler (if any)
+          takes over. This parameter allows to specifies an absolute path where core dump files should be stored until
+          a handler is installed. The path should be absolute and might contain specifiers, see
```

Comment:
I thought that the use of specifiers would have been unlikely hence the use of "might". But after a second thought, it could be that it's not as unlikely as I initially thought so "may" is actually better indeed. Thanks.